README.md

```diff
@@ -0,0 +1,8 @@
+# FlyINN
+
+Intermodal mobility app for a sharing economy.
```

Comment:
Better describe what FlyInn is. IMHO this does not bring across the purpose or functionality.